### PR TITLE
Mobileversion

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
### Hey there!😬😬😬

**Update my portfolio to a mobile version 📱📱**👀🦾

My previous portfolio was in the web version, so I did all again.

I added:

-  images 🖼🎭
- icons 🎎
- my name as a logo 🏴
- contact form 📞☎

 I worked on the redundancy of my project, now my css and html looks cleaner 😎😁✌

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,192 +1,159 @@
...
+           <p>
+            If you have an application you are interested in developing, a
+             feature that you need built or a project that needs coding. 
+             I’d love to help with it 
+           </p>
+           </section>
+
+           <form action="https://formspree.io/f/mayadowe"  method="POST">
+               <ul class="form-footer">
+                   <li><input type="text" class="input-footer" autocomplete="off" required name="name" placeholder="Text Input" maxlength="30"></li>
```
Comment (AshakaE) [inline index.html]: - [x] Please change the placeholder for this input to `name`

File: index.html [hunk 2]
```diff
@@ -1,192 +1,159 @@
...
+                   <li><input type="email" class="input-footer" autocomplete="off" required name="email" placeholder="cfelizmedina@gmail.com" ></li>
```
Comment (AshakaE) [inline index.html]: ### _Optional_
- [x] Please change the placeholder of the email input, this one looks like an actual email. try something like `name@email.com`

File: index.html [hunk 3]
```diff
@@ -1,192 +1,159 @@
...
+                   <li><textarea name="textarea" class="textarea-footer" autocomplete="off" placeholder="Enter text here" maxlength="500"></textarea></li>
```
Comment (AshakaE) [inline index.html]: - [x] Please add the `required` attribute for this textarea input

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @xJokeer 👋  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [x] Please try as much as possible to mimic the design from the template (font size, font color, margins and padding)

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html]: Well resolved

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html]: Well resolved

File: index.html (hunk 3 above)
Comment (rloterh) [reply index.html]: Well resolved

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Elim, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 💯 👍.
Congratulations! 🎉 🎉 🥂


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**